### PR TITLE
Upload image

### PR DESCRIPTION
**Pull request by VadimBlagiy**
finished upload image for Event, little change columns table image, refactored code

### CODE REVIEW

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/ImageUploadServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import java.net.MalformedURLException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+@Qualifier("imageUploadServiceImpl")
```
Comment (ikantsevich) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/ImageUploadServiceImpl.java:27]: It is needed here?

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.05.10_change_column_image_name.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id:  change_column_image_name
+      author:  vadblagiy@gmail.com
+      preConditions:
+        onFail: MARK_RAN
+        tableExists:
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/2021.05.10_change_column_image_name.yaml:7]: please check that column exists, not a table

Comment (VadimBlagiy) [reply sandbox-backend/src/main/resources/db/changelog/changes/2021.05.10_change_column_image_name.yaml:7]: ок

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/ImageUploadServiceImpl.java (hunk 1 above)
Comment (VadimBlagiy) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/ImageUploadServiceImpl.java]: no

**Review by ikantsevich — APPROVED** [review]

**Pull request merged**